libsolidity/inlineasm/AsmParser.cpp

```diff
@@ -347,6 +348,9 @@ Parser::ElementaryOperation Parser::parseElementaryOperation(bool _onlySinglePus
 				if (info.ret != 1)
 					fatalParserError("Instruction \"" + literal + "\" not allowed in this context.");
 			}
+			if (m_flavour == AsmFlavour::Strict)
```

Comment:
This shouldn't be needed as `parseCall` will reject instructions.

Reply:
I think so, too, but I would like to keep it anyways.

Reply:
I'd prefer to have them in a single location, though wouldn't be against an assertion here, that way we can find parser bugs.